unittests/block_header_tests.cpp

```diff
@@ -19,21 +18,21 @@ BOOST_AUTO_TEST_CASE(instant_finality_extension_with_empty_values_test)
 {
    block_header       header;
    constexpr uint32_t last_qc_block_num {0};
-   constexpr bool     is_last_qc_strong {false};
+   constexpr bool     is_strong_qc {false};
```

Comment:
Why this name change? I think the previous one matched `last_qc_block_num` better.

Reply:
Done